src/stores/loginStore.ts

```diff
@@ -32,10 +32,13 @@ export class LoginStore {
         this._isAuthenticated = authRespose.isOk;
         this._hasWriteAccess = authRespose.hasWriteAccess;
         this._userEmail = authRespose.email;
+    }
 
-        // TODO: this is only temporary:
-        // tslint:disable-next-line:no-console
-        console.log(`User: ${authRespose.email} is now authenticated`);
+    @action
+    public clear() {
+        this._isAuthenticated = false;
+        this._userEmail = undefined;
+        this._hasWriteAccess = false;
```

Comment:
Hmm I wouldn't make clear func since it is only called once. I would go with:

constructor() {
      this._isAuthenticated = !Constants.isLoginRequired;
      this._hasWriteAccess = !Constants.isLoginRequired;
      this._userEmail = undefined;
}

Reply:
Lets go with current implementation until we do full logout